spec/might/filter_parameters_spec.rb

```diff
@@ -48,4 +48,18 @@
       end
     end
   end
+
+  describe '#map' do
+    let(:filters) { Might::FilterParameters.new([1,2]) }
+    subject(:mapped) { filters.map { |v| v*2 } }
+
+    it 'returns another object of the same type' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.